tests/beaker_tests/netdev_stdlib/name_server_provider_defaults.rb

```diff
@@ -74,7 +74,7 @@
 
     # Expected exit_code is 0 since this is a vegas shell cmd.
     # Flag is set to true to check for absence of RegExp pattern in stdout.
-    cmd_str = UtilityLib.get_vshell_cmd('show running-config section name-server')
+    cmd_str = UtilityLib.get_vshell_cmd('show running-config | section name-server')
```

Comment:
We should still remove it and replace it with an ensure => absent check similar to how radius_server tests.  See the following.

https://github.com/cisco/cisco-network-puppet-module/blob/develop/tests/beaker_tests/radius_server/radius_server_provider_defaults.rb#L70